examples/agb/collection.json

```diff
@@ -0,0 +1,90 @@
+{
+  "type": "Collection",
+  "id": "icesat2-boreal-v2.1-agb",
+  "stac_version": "1.1.0",
+  "description": "Gridded predictions of aboveground biomass (Mg/ha) for boreal region derived from ICESat-2 and Harmonized Landsat Sentinel 2 data.",
+  "links": [
+    {
+      "rel": "root",
+      "href": "./collection.json",
+      "type": "application/json",
+      "title": "Icesat2 Boreal v2.1: Gridded Aboveground Biomass Density"
+    },
+    {
+      "rel": "predecessor-version",
+      "href": "https://stac.maap-project.org/collections/icesat2-boreal",
+      "title": "Previous version"
+    },
+    {
+      "rel": "item",
+      "href": "./boreal_agb_2020_202411251732556086_0000004/boreal_agb_2020_202411251732556086_0000004.json",
+      "type": "application/geo+json"
+    }
+  ],
+  "stac_extensions": [
+    "https://stac-extensions.github.io/version/v1.2.0/schema.json",
+    "https://stac-extensions.github.io/render/v2.0.0/schema.json"
+  ],
+  "item_assets": {
+    "cog": {
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "roles": [
+        "data"
+      ],
+      "title": "Gridded predictions of aboveground biomass (Mg/ha)",
+      "description": "Gridded predictions of aboveground biomass (Mg/ha)"
+    },
+    "training_data_csv": {
+      "type": "text/csv",
+      "roles": [
+        "data"
+      ],
+      "title": "Tabular training data",
+      "description": "Tabular training data with latitude, longitude, and biomass observations"
+    }
+  },
+  "version": "v2.1",
+  "deprecated": false,
+  "renders": {
+    "agb": {
+      "title": "Aboveground biomass (Mg/ha)",
+      "assets": [
+        "cog"
+      ],
+      "expression": "cog_b1",
```

Comment:
I think we don't need `assets` if we include `expression`. I would have done `assets=cog` and `bidx=1` but that wasn't working in the version of titiler that I have running locally. `expression=cog_b1` does work, though.